internal/info/info_bench_test.go

```diff
@@ -0,0 +1,119 @@
+package info
+
+import (
+	"testing"
+)
+
+func BenchmarkString(b *testing.B) {
+	Init("benchmark")
+	b.ReportAllocs()
+	b.ResetTimer()
+	b.RunParallel(func(pb *testing.PB) {
+		for pb.Next() {
+			if s := String(); s == "" {
+				b.Error("String return empty string")
+			}
+		}
+	})
```

Comment:
Should we clear sync.once method with teardown? 🤔 
I think `BenchmarkString` also use the once method.

Reply:
I think it is not needed. This is the benchmark for string method, there is nothing to do with sync.once